pkgs/development/python-modules/drf-nested-routers/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools
+, django
+, djangorestframework
+}:
+
+buildPythonPackage rec {
+  pname = "drf-nested-routers";
+  version = "0.92.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0rizimr0sxzspc8gn1fd2z1bp62x0wkkr29vwr4nbcg96dik4slx";
+  };
+
+  propagatedBuildInputs = [ django djangorestframework setuptools ];
+
+  # django.core.exceptions.ImproperlyConfigured: Requested setting REST_FRAMEWORK, but settings are not configured. You must either define the environment variable DJANGO_SETTINGS_MODULE or call settings.configure() before accessing settings.
+  doCheck = false;
```

Comment:
Execute the tests by running the command given here: https://github.com/alanjds/drf-nested-routers/blob/master/tox.ini#L13

Reply:
I noticed that `runtests.py` & `tests/` aren't included in the published package.
The only published tests are in [`rest_framework_nested/runtests/`](https://github.com/alanjds/drf-nested-routers/tree/master/rest_framework_nested/runtests) and they're failing without any helpful error message:
```
builder for '/nix/store/gbfv4g234myqc595d7w9rspgmkjadqhn-python3.8-drf-nested-routers-0.92.5.drv' failed with exit code 1; last 10 log lines:
  Executing pythonRemoveTestsDir
  Finished executing pythonRemoveTestsDir
  running install tests
  > /build/drf-nested-routers-0.92.5/rest_framework_nested/runtests/runtests.py(33)main()
       32
  ---> 33     test_runner = TestRunner()
       34     if len(sys.argv) == 2:

  ipdb>
  Exiting Debugger.
```

Reply:
When tests aren't included we usually fetch the Git repository as `src`, because tests are a really valuable metric, especially during updates.

Reply:
I just did that and the tests are running & succeeding now. Thanks!